frontend/src/basic/RobotPage/RobotProfile.tsx

```diff
@@ -154,7 +154,7 @@ const RobotProfile = ({
           )}
         </Grid>
 
-        {loadingCoordinators > 0 ? (
+        {loadingCoordinators > 0 && !Boolean(robot?.activeOrderId) ? (
```

Comment:
Once the order is found it's not necessary to continue displaying this to the user